lib/mixpanel-ruby/error.rb

```diff
@@ -0,0 +1,45 @@
+module Mixpanel
+
+  # Mixpanel specific errors that are thrown in the gem.
+  # In the default consumer we catch all errors and raise
+  # Mixpanel specific errors that can be handled in a custom
+  # Error handler.
+  class MixpanelError < StandardError
+  end
+  
+  class ConnectionError < MixpanelError
+  end
+  
+  class ServerError < MixpanelError
+  end
+
+
+  # The default behavior of the gem is to silence all errors
```

Comment:
is it worth adding this block to the docs somewhere? people may not discover it here